.github/workflows/build_and_test.yml

```diff
@@ -23,6 +23,8 @@ env:
   # we build a dev binary for use in CI so skip downloading
   # canary next-swc binaries in the monorepo
   NEXT_SKIP_NATIVE_POSTINSTALL: 1
+  DATADOG_API_KEY: ${{ secrets.DATA_DOG_API_KEY }}
```

Comment:
Is this a typo? One uses `DATADOG` and the other uses `DATA_DOG`

Reply:
no, datadog's de facto recommendataion is set env to `DATADOG_API_KEY` (which try to read it by default), and in our gh secret settings we stored key with `DATA_DOG_API_KEY`. So in here we coerce its value.